package.json

```
@@ -27,8 +27,16 @@
 		"lint:js": "eslint",
 		"lint": "npm run typecheck:api && npm run typecheck:client && npm run lint:js",
 
-		"test": "vitest",
-		"coverage": "vitest run --coverage"
+		"test:unit": "vitest",
+		"coverage": "vitest run --coverage",
+
+		"test:api:locally": "hurl --variables-file .dev.vars api/src/routes/**/*.hurl --verbose --test",
+		"test:api:ci": "hurl --glob 'tests/**/*.hurl' --continue-on-error --test --jobs 4 --max-time 300 --no-color --output-tap results.tap --variable-file .env.vars",
+		"test:api:auth1": "hurl --variables-file .dev.vars api/src/routes/auth/01_sign-up.hurl --test --verbose",
+		"test:api:auth2": "hurl --variables-file .dev.vars api/src/routes/auth/02_signin-signout-heartbeat.hurl --test --verbose",
+		"test:api:profiles": "hurl --variables-file .dev.vars api/src/routes/profile/03_profile.hurl --test --verbose",
+		"test:api:teams": "hurl --variables-file .dev.vars api/src/routes/team/04_teams.hurl --test --verbose",
+		"test:api:team-members": "hurl --variables-file .dev.vars api/src/routes/team-members/05_team-members.hurl --test --verbose"
```

Try this:
```suggestion
		"test:api:file": "hurl --variables-file .dev.vars --test --verbose"
```

You can pass the other argument to npm like common unix tools:
```sh
npm run test:api:file -- path/to/file
```